src/Imath/ImathInt64.h

```diff
@@ -3,28 +3,34 @@
 // Copyright Contributors to the OpenEXR Project.
 //
 
+///
+/// @file  ImathInt64.h
+///
+/// @brief 64-bit integer types
+///
+
 #ifndef INCLUDED_IMATH_INT64_H
 #define INCLUDED_IMATH_INT64_H
 
-//----------------------------------------------------------------------------
-//
-//	Int64 -- unsigned 64-bit integers
-//
-//----------------------------------------------------------------------------
-
 #include "ImathNamespace.h"
 #include <limits.h>
 
 IMATH_INTERNAL_NAMESPACE_HEADER_ENTER
 
 #if (defined _WIN32 || defined _WIN64) && _MSC_VER >= 1300
+/// Int64 - unsigned 64-bit integer
 typedef unsigned __int64 Int64;
+/// SInt64 - signed 64-bit integer
 typedef __int64 SInt64;
 #elif ULONG_MAX == 18446744073709551615LU
+/// Int64 - unsigned 64-bit integer
 typedef long unsigned int Int64;
+/// SInt64 - signed 64-bit integer
 typedef long int SInt64;
 #else
+/// Int64 - unsigned 64-bit integer
 typedef long long unsigned int Int64;
+/// SInt64 - signed 64-bit integer
 typedef long long int SInt64;
 #endif
```

Comment:
Aside -- I can't quite remember, why do we still have these instead of using stdint?

Reply:
This is worth a bigger discussion, can we eliminate this type declaration altogether? At the very least, the names are inconsistent and confusing. But they're used throughout OpenEXR.